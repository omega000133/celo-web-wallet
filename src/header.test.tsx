import { render } from "@testing-library/react";
import * as React from "react";
import Header from "./header";

describe(Header, () => {
  test("snapshot", () => {
    const { asFragment } = render(<Header />);
    expect(asFragment()).toMatchSnapshot();
  });
});
