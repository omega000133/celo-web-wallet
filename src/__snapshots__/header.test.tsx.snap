// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header snapshot 1`] = `
<DocumentFragment>
  <h1
    class="css-pm4pww-Header"
  >
    Bye
  </h1>
</DocumentFragment>
`;
